src/components/RecipientSidePanel.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Box } from '@chakra-ui/react';
+
+export default function RecipientSidePanel() {
+  return (
+    <div style={{ paddingTop: '20px', paddingLeft: '40px' }}>
```

Comment:
Nit: use `Box` here instead of `div` and replace `style` with chakra ui CSS props